src/sentry/seer/similarity/utils.py

```diff
@@ -179,18 +179,22 @@ def get_stacktrace_string(data: dict[str, Any]) -> str:
     html_frame_count = 0  # for a temporary metric
     stacktrace_str = ""
     found_non_snipped_context_line = False
+    is_frames_truncated = False
```

Comment:
Nit:

```suggestion
    are_frames_truncated = False
```

or

```suggestion
    were_frames_truncated = False
```

or 

```suggestion
    is_truncated_stacktrace = False
```

etc.